client_ext_test.go

```diff
@@ -98,6 +98,7 @@ func TestClientPeer(t *testing.T) {
 			assert.Nil(t, closeErr)
 		})
 		assert.NotNil(t, clientStream.Peer().Addr)
+		assert.NotNil(t, clientStream.Peer().Protocol)
```

Comment:
```suggestion
		assert.NotEmpty(t, clientStream.Peer().Addr)
		assert.NotEmpty(t, clientStream.Peer().Protocol)
```
Addr + Protocol are strings so `assert.NotEmpty` is never going to fail